src/exportNameRule.ts

```diff
@@ -162,7 +157,7 @@ export class ExportNameWalker extends Lint.RuleWalker {
 
     private validateExport(exportedName: string, node: ts.Node): void {
         const flags = Rule.getIgnoreCase(this.getOptions()) ? 'i' : '';
-        const regex : RegExp = new RegExp(exportedName + '\..*', flags); // filename must be exported name plus any extension
+        const regex: RegExp = new RegExp(exportedName + '..*', flags); // filename must be exported name plus any extension
```

Comment:
interesting change, may be there was an error and it should be `\\..`

Reply:
Should I make it `\\`? This change was made by prettier.